test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14107,6 +14107,31 @@ def enable_cgroup_cset(self):
                               self.hostname)
             raise AssertionError('cgroup subsystems not mounted')
 
+    def cleanup_homedir(self, user):
```

Comment:
Add a spynx header for this function.

Reply:
I think right place for this clean code is in runner plugin where we already have other cleanup code. https://github.com/openpbs/openpbs/blob/master/test/fw/ptl/utils/plugins/ptl_test_runner.py#L987